### DISCUSSION TITLE
How to make MkDocs Material interactive?

### DISCUSSION THREAD

**Discussion by oceanumeric** [opengeos/leafmap#295] [Q&A] [answered]
Hey,

Can I ask how did you make MkDocs Material interactive for your maps?

Thanks in advance :)

**Comment by giswqs** [comment, marked as answer]
Use `import leafmap.foliumap as leafmap` or set "USE_FOLIUM" as an environment variable, which will be the folium back to create maps. Folium maps works with Mkdocs, while ipyleaflet does not.
